cmake/external/onnxruntime_external_deps.cmake

```diff
@@ -656,11 +656,16 @@ if (onnxruntime_USE_WEBGPU)
 
     # Vulkan may optionally be included in a Windows build. Exclude until we have an explicit use case that requires it.
     set(DAWN_ENABLE_VULKAN OFF CACHE BOOL "" FORCE)
+    # We are currently always using the D3D12 backend.
+    set(DAWN_ENABLE_D3D11 OFF CACHE BOOL "" FORCE)
```

Comment:
Do the options we set here have impact on DawnProcTable's size?

Reply:
No. The proc table is generated by a [template](https://github.com/google/dawn/blob/main/generator/templates/dawn_proc_table.h) using [dawn.json](https://github.com/google/dawn/blob/main/src/dawn/dawn.json) as data.